gui/public/i18n/ja/translation.ftl

```diff
@@ -174,29 +174,29 @@ body_assignment_menu-unassign_tracker = トラッカーの割り当て解除
 #
 # We are using it here because english doesn't require changing the text in each case but
 # maybe your language does.
--tracker_selection-part = Which tracker to assign to your
-tracker_selection_menu-NONE = Which tracker do you want to be unassigned?
-tracker_selection_menu-HEAD = { -tracker_selection-part } head?
-tracker_selection_menu-NECK = { -tracker_selection-part } neck?
-tracker_selection_menu-RIGHT_SHOULDER = { -tracker_selection-part } right shoulder?
-tracker_selection_menu-RIGHT_UPPER_ARM = { -tracker_selection-part } right upper arm?
-tracker_selection_menu-RIGHT_LOWER_ARM = { -tracker_selection-part } right lower arm?
-tracker_selection_menu-RIGHT_HAND = { -tracker_selection-part } right hand?
-tracker_selection_menu-RIGHT_UPPER_LEG = { -tracker_selection-part } right thigh?
-tracker_selection_menu-RIGHT_LOWER_LEG = { -tracker_selection-part } right ankle?
-tracker_selection_menu-RIGHT_FOOT = { -tracker_selection-part } right foot?
-tracker_selection_menu-RIGHT_CONTROLLER = { -tracker_selection-part } right controller?
-tracker_selection_menu-CHEST = { -tracker_selection-part } chest?
-tracker_selection_menu-WAIST = { -tracker_selection-part } waist?
-tracker_selection_menu-HIP = { -tracker_selection-part } hip?
-tracker_selection_menu-LEFT_SHOULDER = { -tracker_selection-part } left shoulder?
-tracker_selection_menu-LEFT_UPPER_ARM = { -tracker_selection-part } left upper arm?
-tracker_selection_menu-LEFT_LOWER_ARM = { -tracker_selection-part } left lower arm?
-tracker_selection_menu-LEFT_HAND = { -tracker_selection-part } left hand?
-tracker_selection_menu-LEFT_UPPER_LEG = { -tracker_selection-part } left thigh?
-tracker_selection_menu-LEFT_LOWER_LEG = { -tracker_selection-part } left ankle?
-tracker_selection_menu-LEFT_FOOT = { -tracker_selection-part } left foot?
-tracker_selection_menu-LEFT_CONTROLLER = { -tracker_selection-part } left controller?
+-tracker_selection-part = どのトラッカーに割り当てるか？
```

Comment:
Ok, let me explain this one.
``-tracker_selection-part`` is a term, we can't use it in the slimevr code. It's made so translators don't need to repeat themselves a lot of times for the same thing.

Currently your text looks like this: ``どのトラッカーに割り当てるか？ 右手?``
An alternative would be to do something like this:
```ftl
-tracker_selection-part = どのトラッカーを{ $body-part }に割り当てるか？
tracker_selection_menu-HEAD = { -tracker_selection-part(body-part: "頭") }
tracker_selection_menu-RIGHT_HAND = { -tracker_selection-part(body-part: "右手") }
```
and it would look like this:
``どのトラッカーを頭に割り当てるか？``
``どのトラッカーを右手に割り当てるか？``

You don't need to do it this way or use the same text I just gave you. Just giving you an example.
You could also remove ``-tracker_selection-part`` and do ``tracker_selection_menu-HEAD=どのトラッカーを頭に割り当てるか？``